obspy/core/util/headers.py

```diff
@@ -0,0 +1,484 @@
+# -*- coding: utf-8 -*-
+"""
+Base constants for ObsPy.
+
+:copyright:
+    The ObsPy Development Team (devs@obspy.org)
+:license:
+    GNU Lesser General Public License, Version 3
+    (https://www.gnu.org/copyleft/lesser.html)
+"""
+# NO IMPORTS FROM OBSPY OR FUTURE IN THIS FILE! (file gets used at
+# installation time)
+import sys
+
+
+# XXX TODO when bumping to numpy 1.9.0: replace bytes() in io.reftek with
+# np.tobytes()
+HARD_DEPENDENCIES = {
+    "future": ">=0.12.4",
+    "numpy": ">=1.6.1",
+    "scipy": ">=0.9.0",
+    "matplotlib": ">=1.1.0",
+    "lxml": "",
+    "setuptools": "",
+    "sqlalchemy": "",
+    "decorator": "",
+    "requests": ""
+    }
+OPTIONAL_DEPENDENCIES = {
+    'arclink': {
+        "cryptography": "",
+        "m2crypto": "",
+        "pycrypto": ""},
+    # for docs, also see conda environment file under
+    # misc/docs/py3-docs-master.yml
+    # other versions might work as well but these are currently used in our
+    # docs buildbot and therefore confirmed to work
+    'docs': {
+        "sphinx": "==1.2.2",
```

Comment:
Not so sure we should hard-code these. Makes it annoying to work with other versions.